src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE users(
+    id bigint GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    first_name varchar(50) NOT NULL,
+    last_name varchar (50),
+    email varchar (255) UNIQUE NOT NULL,
+    password varchar (50) NOT NULL,
+    phone varchar (50),
+    created_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP,
+    deleted_at TIMESTAMP
+);
+
+CREATE TABLE user_roles(
+    id int GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    role varchar (50) NOT NULL,
+    user_id int,
```

Comment:
The `ID` column in the `users` table has the type `BIGINT`, but it's still `INT` here.

Reply:
Whoops! Forgot to check it here :)